build-frontend/action.yml

```diff
@@ -52,27 +55,58 @@ runs:
         username: ${{ inputs.cloud_username }}
         password: ${{ inputs.cloud_password }}
 
-    - name: Resolve values of the versions, docker ref & dockerfile path
+    - name: Resolve docker buildargs
       id: resolve
-      shell: bash
-      run: |
-        # Set the version of the app based on user input or use git commit hash
-        echo "version=${{ inputs.version || github.sha }}" >> $GITHUB_OUTPUT
-        # Set the reference address for app in docker registry
-        echo "ref=${{ inputs.docker_registry_url }}/${{ inputs.project_name }}-storefrontcloud-io" >> $GITHUB_OUTPUT
-        # Set the path to the Dockerfile base on selected frontend framework
-        if [[ $FRONTEND == "next" ]]; then
-          echo "path=.vuestorefrontcloud/docker/nextjs/Dockerfile-frontend" >> $GITHUB_OUTPUT
-        elif [[ $FRONTEND == "nextjs" ]]; then
-          echo "path=.vuestorefrontcloud/docker/nextjs/Dockerfile-frontend" >> $GITHUB_OUTPUT
-        elif [[ $FRONTEND == "nuxt" ]]; then
-          echo "path=.vuestorefrontcloud/docker/nuxtjs/Dockerfile-frontend" >> $GITHUB_OUTPUT
-        else
-          echo "Error: Unsupported FRONTEND value. Valid options are 'next' or 'nuxt'."
-          exit 1
-        fi
+      uses: actions/github-script@v7
       env:
-        FRONTEND: ${{ inputs.frontend }}
+        INPUT_NPM_EMAIL: ${{ inputs.npm_email }}
+        INPUT_NPM_PASS: ${{ inputs.npm_pass }}
+        INPUT_NPM_USER: ${{ inputs.npm_user }}
+        INPUT_NPM_REGISTRY: ${{ inputs.npm_registry }}
+        INPUT_IMAGE_PROVIDER: ${{ inputs.image_provider }}
+        INPUT_DOCKER_BUILDARGS: ${{ inputs.docker_buildargs }}
+        INPUT_FRONTEND: ${{ inputs.frontend }}
+        INPUT_VERSION: ${{ inputs.version }}
+        INPUT_DOCKER_REGISTRY_URL: ${{ inputs.docker_registry_url }}
+        INPUT_PROJECT_NAME: ${{ inputs.project_name }}
+      with:
+        script: |
+          const base = {
+            'NPM_EMAIL': core.getInput('npm_email'),
+            'NPM_PASS': core.getInput('npm_pass'),
+            'NPM_USER': core.getInput('npm_user'),
+            'NPM_REGISTRY': core.getInput('npm_registry'),
+            'NUXT_IMAGE_PROVIDER': core.getInput('image_provider'),
+          };
+
+          const transformedBaseArgs = Object.entries(base)
+            .map(([key, value]) => `${key}=${value}`);
+
+          const extraArgs = core.getMultilineInput('docker_buildargs', { required: false });
+          const mergedArgs = [
+            ...transformedBaseArgs
+            ...extraArgs,
+          ];
```

Comment:
I am almost sure this case is tested, but ask anyway - what is the result of empty `getMultilineInput` input?
`...extraArgs` wont throw error when empty?